api/apps/geoprocessing/src/modules/scenario-planning-units-inclusion/__mocks__/include-sample.ts

```diff
@@ -53,6 +53,62 @@ export const excludeSample = (): FeatureCollection<Polygon | MultiPolygon> => ({
 });
 
 export const includeSample = (): FeatureCollection<Polygon | MultiPolygon> => ({
+  type: 'FeatureCollection',
```

Comment:
Please update the image related to sample, otherwise it will be really hard to understand what changes/is going on.

Reply:
good point. 501d1145f046337c25a92b37cafc27fc7f9a438c